src/design-system/components/input-field/InputField.tsx

```diff
@@ -1,5 +1,5 @@
 import '~scss-components/input-field/InputField.scss'
-import { InputFieldVariantEnum } from '~/types'
+import { InputFieldVariantEnum } from './InputField.constants'
```

Comment:
Move this import to the bottom of imports. It's an excellent practice when all imports are not chaotically sorted.
Firstly - all imports from external libs, after that - imports from global packages in our project, and last ones - from neighboring files.
Unfortunately, we don't have a linter for that, but I think we should add it in the future.

Reply:
Fixed!)